packages/firebase_storage/test/firebase_storage_test.dart

```diff
@@ -52,5 +52,39 @@ void main() {
             equals(new Uint8List.fromList(<int>[1, 2, 3, 4])));
       });
     });
+
+    group('deleteFile', () {
+      const MethodChannel channel = const MethodChannel(
+        'firebase_storage',
+      );
+
+      final List<MethodCall> log = <MethodCall>[];
+
+      StorageReference ref;
+
+      setUp(() {
+        channel.setMockMethodCallHandler((MethodCall methodCall) {
+          log.add(methodCall);
+          return new Future<bool>.value(true);
```

Comment:
[nit] The actual method invocation will return a future that completes with `null`, not `true`. You can do that with less syntax using the `async` keyword:
```dart
channel.setMockMethodCallHandler((MethodCall methodCall) async {
  log.add(methodCall);
  return null;
});
```
(If you feel up to it, the test for `getData` could be simplified similarly).

Reply:
Thanks! It's a left over from DeleteTask result, sorry about that.